src/metrics_server/README.md

```diff
@@ -21,6 +21,7 @@ The metrics server supports two URL paths:
     endUtcMs: number,
     userReports: [{
         countries: string[],
+        asn: string[],
```

Comment:
Each entry should have only one ASN. Otherwise analysis becomes a major pain.

Reply:
Done.